bin/nvcc_wrapper

```diff
@@ -196,30 +197,28 @@ do
   #Handle c++11
   --std=c++1y|-std=c++1y|--std=c++1z|-std=c++1z|--std=gnu++1y|-std=gnu++1y|--std=gnu++1z|-std=gnu++1z)
     if [ $stdcxx_applied -eq 1 ]; then
-       echo "nvcc_wrapper - *warning* you have set multiple optimization flags (-std=c++1* or --std=c++1*), only the first is used because nvcc can only accept a single std setting"
+       warn_std_flag
     else
-       if [ ! -z "${KOKKOS_CMAKE_CXX_STANDARD}" ]; then
-         echo "nvcc_wrapper does not accept intermediate standard flag $1, but I will use -std=c++${KOKKOS_CMAKE_CXX_STANDARD} instead"
-         shared_args="$shared_args -std=c++${KOKKOS_CMAKE_CXX_STANDARD}"
-       else
-	echo "nvcc_wrapper has been given intermediate standard flag $1 - nvcc only accepts full standard flags like -std=c++11"
-	exit 1
-       fi
+       # this is hopefully just occurring in a downstream project during CMake feature tests
+       # we really have no choice here but to accept the flag and "promote" to the corresponding C++ standard
+       echo "nvcc_wrapper does not accept intermediate standard flag $1, but I will use -std=c++14 instead. It is undefined behavior to use this flag. This should only be occurring during CMake configuration."
+       shared_args="$shared_args -std=c++14"
```

Comment:
Why aren't we emitting that warning unconditionally (regardless of whether multiple cxx standard flags are passed or not)?

Reply:
The second flag is ignored. But you would prefer that we warn (or error) when an ignored flag is also invalid?